test/Microsoft.Sbom.Targets.Tests/AbstractGenerateSBomTaskInputTests.cs

```diff
@@ -226,8 +256,8 @@ public void Sbom_Generation_Succeeds_For_Null_Verbosity()
     public void Sbom_Generation_Succeeds_For_Invalid_Verbosity()
     {
         // Arrange
-        // If an invalid Verbosity is specified, the default value should be Verbose and is printed in the
-        // tool's standard output.
+        // If an invalid Verbosity is specified, the default value should be Verbose. It is also printed in the
+        // tool's standard output for the MSBuild Core task.
         var pattern = new Regex("Verbosity=.*Value=Verbose");
```

Comment:
We should update this to "Information" as we changed the default verbosity

Reply:
Good catch. Fixed at 1e50b1a3bb7c3df73bb2d192f3bdb7ad9ce7e40e